packages/cli/src/commands/new.ts

```diff
@@ -26,10 +26,9 @@ export default class New extends Command {
 
   static flags = {
     help: flags.help({char: 'h'}),
-    ts: flags.boolean({
-      description: 'generate a TypeScript project',
-      default: true,
-      allowNo: true,
+    js: flags.boolean({
+      description: 'Generates a JS project. Omitting this flag will TypeScript as the default language.',
```

Comment:
```suggestion
      description: 'Generates a JS project. TypeScript is the default unless you add this flag.',
```

I went to fix a typo and decided to tweak it a bit